src/ParallelTestRunner.jl

```diff
@@ -780,16 +794,16 @@ function runtests(mod::Module, ARGS; test_filter = Returns(true), RecordType = T
                     wrkr = something(test_worker(test), p)
 
                     test_t0 = time()
-                    running_tests[test] = (wrkr, test_t0)
+                    running_tests[test] = (worker_id(wrkr), test_t0)
 
                     test, wrkr, test_t0
                 end
 
                 # run the test
-                put!(printer_channel, (:started, test, wrkr))
+                put!(printer_channel, (:started, test, worker_id(wrkr)))
                 result = try
-                    Distributed.remotecall_eval(Main, wrkr, :(import ParallelTestRunner))
-                    remotecall_fetch(runtest, wrkr, RecordType, test_runners[test], test,
+                    Malt.remote_eval_wait(Main, wrkr, :(import ParallelTestRunner))
+                    Malt.remote_call_fetch(invokelatest, wrkr, runtest, RecordType, test_runners[test], test,
```

Comment:
Why is the `invokelatest` required? I ran into the same when trying out Malt yesterday. `@invokelatest Malt.remote_call_fetch` also worked.

Reply:
Malt and Distributed start a Task on the worker that deserialize messages from the main process and then execute the referenced function. This Task freezes the world-age it executes in and you require at least one invokelatest to execute new functions. 

Malt doesn't have the invokelatest in its Task and seems to rely on the user to manage that. 

https://github.com/JuliaLang/Distributed.jl/blob/661112ef29b8d371b8612b94a629c0bbcf3d36f7/src/process_messages.jl#L283

Reply:
Thanks.